src/Tribe/RSVP.php

```diff
@@ -2266,33 +2645,47 @@ public function generate_tickets_for( $product_id, $ticket_qty, $attendee_detail
 	public function parse_attendee_details() {
 		$order_id = self::generate_order_id();
 
-		$attendee_email     = empty( $_POST['attendee']['email'] ) ? null : sanitize_email( $_POST['attendee']['email'] );
-		$attendee_email     = is_email( $attendee_email ) ? $attendee_email : null;
-		$attendee_full_name = empty( $_POST['attendee']['full_name'] ) ? null : sanitize_text_field( $_POST['attendee']['full_name'] );
-		$attendee_optout    = empty( $_POST['attendee']['optout'] ) ? 0 : $_POST['attendee']['optout'];
+		$first_attendee = [];
+
+		if ( ! empty( $_POST['tribe_tickets'] ) ) {
+			$first_ticket = current( $_POST['tribe_tickets'] );
+
+			if ( ! empty( $first_ticket['attendees'] ) ) {
+				$first_attendee = current( $first_ticket['attendees'] );
+			}
+		} elseif ( isset( $_POST['attendee'] ) ) {
+			$first_attendee = $_POST['attendee'];
```

Comment:
* :warning: Detected usage of a non-sanitized input variable: $_POST['attendee']
* :no_entry_sign: Processing form data without nonce verification.

<sub>hash: 7bb79ac4e2ba7741acdf366d75be7b8f</sub>